app/src/main/java/com/example/seizuredetectionapp/LoginTabFragment.java

```diff
@@ -77,7 +77,7 @@ public void onClick(View v){
                 // Change to the Datatable page
                 boolean flag = loginUser();
                 if(flag) {
-                    startActivity(new Intent(this.getContext(), QuestionnairePersonal.class));
+                    startActivity(new Intent(this.getContext(), QuestionnaireMedical.class));
```

Comment:
Need to fix. I had changed it to datatable for my testing purposes and incorrrectly changed it back to the wrong activity.